translator/defaultKeyCase.go

```diff
@@ -95,10 +93,10 @@ func DefaultLogGroupClassCase(key string, defaultVal, input interface{}) (return
 		//CreateLogGroup API only accepts values STANDARD or INFREQUENT_ACCESS
 		returnVal = strings.ToUpper(classVal)
 	} else {
-		returnVal = util.StandardLogGroupClass
 		AddErrorMessages(
 			fmt.Sprintf("LogGroupClass key: %s", key),
 			fmt.Sprintf("%s value (%v) in is not a valid Log Group Class field. Defaulting to standard.", key, returnVal))
```

Comment:
We should change this log which says defaulting to standard and change it to Defaulting to empty

Reply:
Yes so the agent wouldn't be sending a request with `LogGroupClass` parameter in this instance, but thinking from a customer perspective, the log group would be created with the default value, which is Standard. If a customer were to go through logs, it might cause more confusion if they see that it's empty